service/raft/raft_group0.cc

```diff
@@ -36,8 +36,10 @@ raft_group0::raft_group0(seastar::abort_source& abort_source,
         gms::gossiper& gs,
         cql3::query_processor& qp,
         service::migration_manager& mm,
-        raft_group0_client& client)
+        raft_group0_client& client,
+        size_t max_record_size)
```

Comment:
Nit: a dedicated struct for storing such configuration parameters would be good.
Tt makes the call-site more readable (instead of passing a magic number `x`, you pass something like `config{.max_record_size=x}`), and it prepares us for potential additional params in the future.

Reply:
Instead of passing it from the top, when initializing the group instance, we could try to pull it from raft::persistence, aka raft::sys_table_storage,  which, via query_processor -> storage_proxy -> database, has a reference to the commit log as well.

Reply:
pulled it from `query_processor -> storage_proxy -> database`